application/app.py

```diff
@@ -157,34 +155,45 @@ def game_status():
 @app.route("/api/game/start", methods=["GET"])
 @requires_auth
 def begin_game():
+    """This endpoint is for once a player is ready to start. Will begin a game if all players (3+) 
+    are ready or there are 7 players
+    Inputs - player_id
+    Outputs -
+        Game not started - status comment
+        Game started - status comment, game (player, cards)"""
     player_id = request.args.get('player_id')
     player = Player.query.filter_by(id=player_id).first()
     player.ready = True
-    db.session.add(player)
 
     try:
+        db.session.add(player)
         db.session.commit()
     except Exception as e:
         print(e)
         return jsonify(message="There was an error"), 501
 
-    # print("Player: " + player.id + " Ready: " + player.ready)
+    ############################
+    # Checks if game can begin #
+    ############################
     players = Player.query.filter_by(gameId=player.gameId).order_by(id).all()
-    print(players);
-    # IF EVERY PLAYER IS READY, AND THERE ARE AT LEAST 3, THEN THE GAME STARTS, OTHERWISE THE GAME WAITS:
     if len(players) > 2:
-        for p in players:
-            if p.ready == False:
-                print("Players not ready")
-                return jsonify(status="Waiting")
+        if len(players) < 7:
```

Comment:
Just a small suggestion here we probably don't need to calculate the length of players twice.
```
player_count = len(players)
if (player_count > 2 and player_count < 7):
```